### PR TITLE
chore(instrumentation-graphql): turn off failing test and add CTA for becoming an owner

### PR DESCRIPTION
**Pull request by Rauno56**
## Which problem is this PR solving?

Our CI is currently failing. I'm skipping the failing test.
Created https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1184 to fix.

## Short description of the changes

1. Turn off failing test
1. Pause releases
1. Add a disclaimer and CTA to the top of the readme

### CODE REVIEW

**Review by pichlermarc — APPROVED** [review]

**Comment by Rauno56** [comment]
@pichlermarc, we had a chat with @blumamir about this and decided **not** to disable it from releasing for now, because:

1. Having it unreleased might cause us issues with metapackages, **if** `@opentelemetry/instrumentation` package is updated.
2. The bug is not so serious to actually cause issues for people using it. The data might not be what they expect(or it might, if the test is invalid), but they might not even notice.

**Comment by Rauno56** [comment]
Also changed the wording in the readme. I'd love if a native speaker could(or anyone that has more confidence in their English than I do) review it.

**Review by blumamir — APPROVED** [review]
LGTM

**Review by pichlermarc — APPROVED** [review]
@Rauno56 I see, thanks for the update. :slightly_smiling_face: 
I think the title of the PR needs to be updated, too :slightly_smiling_face:

File: plugins/node/opentelemetry-instrumentation-graphql/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # OpenTelemetry Instrumentation GraphQL
 
+:bangbang: You could be a component owner for this package and maintain it to bring it up to quality it's users deserve! Check out [open issues](https://github.com/open-telemetry/opentelemetry-js-contrib/labels/pkg%3Ainstrumentation-graphql) on how to help.
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-graphql/README.md:3]:
```suggestion
:bangbang: You could be a component owner for this package and help maintain the quality its users deserve! Check out [open issues](https://github.com/open-telemetry/opentelemetry-js-contrib/labels/pkg%3Ainstrumentation-graphql) on how to help.
```

**Review by dyladan — APPROVED** [review]

**Comment by dyladan** [comment]
Can you also remove the entry from `.github/component-owners.yml`?

**Comment by Rauno56** [comment]
> Can you also remove the entry from `.github/component-owners.yml`?

I'll leave @obecny a code owner at his request. CTA is for finding help for one of the most popular instrumentations we have. I think we can afford having two names listed there. :)

**Pull request merged**